sdks/go/pkg/beam/core/typex/special.go

```diff
@@ -88,6 +89,23 @@ type PaneInfo struct {
 	Index, NonSpeculativeIndex int64
 }
 
+type Timers struct {
```

Comment:
Why are there two of the almost the same type? Since these are exported, they need their own doc comments at least, but given their near-identical-ness, an additional comment about why we need one and not the other would be valuable.